src/hooks/payment/useFetchPaymentRecord.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { paymentRecordSelector } from '../../selectors/payment';
+
+import { fetchPaymentRecord } from '../../actions/payment';
+
+const DELAY_IN_MS = 3000;
+
+const isPending = paymentRecord => true;
+
+const useFetchPaymentRecord = paymentRecordId => {
```

Comment:
Unused?

Reply:
If it's not yet used, maybe consider removing this part from this PR and add it to the future one that uses it?

Reply:
Yes, it's unused. If it's okay to you, I prefer to leave it to the next PR. I don't see any downside, considering our flow.

Reply:
I've changed my mind. There is a down-side. It's hard to review the code which is not clear for the usecase. I'll update it.